src/shared/translation/translationGerman.json

```
@@ -1575,5 +1575,96 @@
   "tooltip.openSettings": "Einstellungen",
   "warning": "Warnung",
   "warning.offline": "Sie sind offline. Bitte überprüfen Sie Ihre Internetverbindung.",
-  "warning.online": "Sie sind wieder online."
+  "warning.online": "Sie sind wieder online.",
+  "pages.aboutus": "Über uns",
+  "pages.aboutus.introductionHeader": "Wer ist HASKI?",
+  "pages.aboutus.introductionBody": "HASKI ist ein Verbundprojekt der drei bayerischen Hochschulen OTH Regensburg, TH Aschaffenburg & HS Kempten und umfasst ein Team von über 20 Professorinnen, wissenschaftlichen Mitarbeiterinnen, studentischen Hilfskräften und an der Verwaltung beteiligten Personen.",
```

Wir haben bisher glaub keinen einheitlichen Standard bezüglich Gendern. Aber so ist es eher nicht richtig.
```suggestion
  "pages.aboutus.introductionBody": "HASKI ist ein Verbundprojekt der drei bayerischen Hochschulen OTH Regensburg, TH Aschaffenburg & HS Kempten und umfasst ein Team von über 20 ProfessorInnen, wissenschaftlichen MitarbeiterInnen, studentischen Hilfskräften und an der Verwaltung beteiligten Personen.",
```